### ISSUE TITLE
createImageBitmap not working for jpegs

### ISSUE THREAD

**Issue by bodo22** [nwjs/nw.js#7372]
NWJS Versions : `0.44.1`, `0.44.0`, `/live-build/nw44/20200212-232911/ed5e0c1a2/v0.44.2/`
Operating System : Ubuntu 16.04 LTS

### Expected behavior

I expect jpgs to be decoded as such, just like pngs.
### Actual behavior

jpgs are not decoded, but an error is thrown:

`DOMException: The source image could not be decoded.`

### How to reproduce

Just start this package.json with the test.html & the error should throw.
[test.zip](https://github.com/nwjs/nw.js/files/4193767/test.zip)

In `chrome 80.0.3987.87 (Official Build) (64-bit)` and `nw v0.43.6` it is working as expected.

**Comment by bodo22** [comment]
Upon further investigation, the error doesnt seem to affect all jpgs in nwjs & has nothing to do with where the image is loaded from. In chrome, no error is thrown.

Attached is another minimal zip, where a buggy jpg cannot be decoded, creating the follwing devtools output:
![screenshot](https://user-images.githubusercontent.com/7796697/74429872-0836fa80-4e5c-11ea-82a9-ed6b04cf2350.png)

[test-2.zip](https://github.com/nwjs/nw.js/files/4198002/test-2.zip)

**Comment by arudnev** [comment]
Yeah, and html image can be decoded from the same content just fine, it's the bitmap that can not be allocated for some reason (when we hit this defect we temporarily switched from ImageBitmap to HTMLImageElement losing ability to do some processing in workers).

```javascript
async function blobImage(blob) {
  const url = URL.createObjectURL(blob);
  try {
    const image = new Image();
    image.src = url;
    await image.decode();
    return image;
  } finally {
    URL.revokeObjectURL(url);
  }
}

(async () => {
  const srcs = {
    jpg: 'https://upload.wikimedia.org/wikipedia/commons/7/74/Earth_poster_large.jpg',
    jpgBuggy: 'https://picsum.photos/id/237/200/300.jpg',
  }
  Object.entries(srcs).forEach(async ([key, src]) => {
    try {
      console.log(key, 'src', src);
      const res = await fetch(src);
      const blob = await res.blob();
      console.log(key, 'type', blob.type);
      const image = await blobImage(blob);
      console.log(key, 'image', { with: image.naturalWidth, height: image.naturalHeight }, image);
      const bitmap = await createImageBitmap(image);
      console.log(key, 'bitmap', bitmap);
    } catch(err) {
      console.error(key, err);
    }
  });
})()
```
<img width="747" alt="Screenshot 2020-02-13 07 48 02" src="https://user-images.githubusercontent.com/1192686/74436988-5b498700-4e35-11ea-9d4a-e870431043bd.png">

**Comment by bodo22** [comment]
Thanks the infos! we are forced to `createImageBitmap`s in web workers (without making unfeasible changes) ATM. Trying to find out the difference between the 2 jpgs for now...

**Comment by bodo22** [comment]
This issue is fixed in the current nightly build [v0.45.0-alpha9](https://dl.nwjs.io/live-build/nw45/20200216-040707/561ae34e5/v0.45.0-alpha9/)

**Comment by rogerwang** [comment]
close as it's fixed in nw45 branch.
